src/pages/index.js

```diff
@@ -45,30 +73,68 @@ const section = new Section(
 );
 section.renderItems();
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  descriptionSelector: ".profile__description",
+// retrieve use info
+api.getUserInfo().then((data) => {
+  // console.log(data);
+  userInfo.setUserInfo({
+    name: data.name,
+    about: data.about,
+    avatar: data.avatar,
+  });
 });
 
+function handleProfileFormSubmit(inputValues) {
+  userInfo.setUserInfo({
+    name: inputValues.name,
+    about: inputValues.about,
+    avatar: inputValues.avatar,
```

Comment:
You need to update `DOM` only in `then` blocks after a successful response from the server so that when there is an error, it shouldn’t change